substrate/frame/contracts/fixtures/contracts/return_with_data.rs

```diff
@@ -38,6 +38,9 @@ pub extern "C" fn call() {
 		output: [u8],
 	);
 
+	// burn some PoV
+	api::clear_storage_v1(b"");
+
```

Comment:
Why is that here?

Reply:
arg looks like I messed up some merge, will remove

Reply:
actually no,
This is related to the update below in the other test
This make the contract consume some PoV so that these asserts works
(They don't anymore since return_value does not consume PoV anymore)
https://github.com/paritytech/polkadot-sdk/blob/pg/bench_tweaks/substrate/frame/contracts/fixtures/contracts/caller_contract.rs?plain=1#L118

Reply:
It is kind of counter intuitive why `clear_storage` is used for that. It works because in order to write an item, we read its size before. However, just reading something would be more clear.

Reply:
Yes right I can convert to a get_storage and just ignore the value, 
I used it here because I used it in the .wat below where it's slightly easier since clear does not push anything on the stack